code/modules/organs/subtypes/machine.dm

```diff
@@ -7,39 +7,43 @@
 	vital = 1 //because it is now hosting the posibrain
 	max_damage = 50 //made same as arm, since it is not vital
 	min_broken_damage = 30
-	encased = null
+	encased = "support frame"
 
 /obj/item/organ/external/head/ipc/New()
 	robotize("Hephaestus Integrated Limb")
 	..()
 
 /obj/item/organ/external/chest/ipc
 	dislocated = -1
-	encased = null
+	encased = "support frame"
 /obj/item/organ/external/chest/ipc/New()
 	robotize("Hephaestus Integrated Limb")
 	..()
 
 /obj/item/organ/external/groin/ipc
 	dislocated = -1
+	encased = "support frame"
```

Comment:
Minor nitpick, but please add a space after these. Cluttered code like this is bad.